docs/contract/logs.md

```diff
@@ -22,15 +22,13 @@ Version: 0.0
 - **ID field** - _optional_
   - The first string field with the name `id` is the id field.
   - Unique identifier of the log line
-- **Attributes field** - _optional_
-  - The first string field with the name `attributes` is the attributes field.
-  - Other systems may refer to this with other names, for example Loki calls these "labels"
-  - This field represent additional attributes of the log line.
+- **Labels field** - _optional_
+  - The first field with the name `labels` is the labels field.
+  - This field represent additional labels of the log line.
```

Comment:
i recommend adjusting this to not say `labels` in the explanation, because that's very loki/grafana specific.. maybe something like:
```
- This field represents additional information about the log line.
```
WDYT?

Reply:
```suggestion
  - This field represent additional information about the log line.
```

Reply:
Good point